### ISSUE TITLE
Poor documentation

### ISSUE THREAD

**Issue by quux00** [kuroneko/gosqlite3#20]
You should provide better documentation how to use this so that someone not extremely well versed in the Go language can get started with it.  I have no idea how to use this library based on your tests.  A page of how-to would be very helpful.

**Comment by feyeleanor** [comment]
Agreed. I'm currently giving all my projects a refresh to make sure they're solid under 1.03 so when I get to gosqlite3 I'll write a short tutorial. In the meantime if there are particular things which baffle you about the API I'm more than happy to answer questions.
